gems/aws-sdk-dynamodb/spec/client_spec.rb

```diff
@@ -161,14 +161,10 @@ module DynamoDB
 
         it 'observes the :simple_attributes configuration option' do
```

Comment:
Validation gets pushed to call time instead of stub_responses time. I'm not sure if this is better or worse.

Reply:
I'm not certain that works in all cases, such as if the stub was a proc, you'd have to resolve it then.